package.json

```diff
@@ -95,5 +62,28 @@
       "path": "./node_modules/cz-conventional-changelog"
     }
   },
-  "version": "0.0.0-development"
+  "pre-commit": "pre-commit",
```

Comment:
Should we switch to husky to remove this boilerplate?

Reply:
I wanted to, but do you have some reasons why you installed pre-commit for this project?

Reply:
At the time I didn't know about husky and didn't want to have all hooks setup. But now I'm sold on husky because of less boilerplate.

Reply:
Oh yeah, if we use husky, we should also rename our `commit` script to be something else..

is `"cz": "git-cz"` okay?